libraries/c_sdk/standard/mqtt/src/iot_mqtt_serializer_deserializer_wrapper.c

```diff
@@ -396,7 +416,7 @@ IotMqttError_t _IotMqtt_subscribeSerializeWrapper( const IotMqttSubscription_t *
     }
 
     /* Free allocated memory as the packet was serialized. */
-    IotMqtt_FreeMessage( subscriptionList );
+    IotMqtt_FreeMessage(subscriptionList);
```

Comment:
Uncrustify. Are you using Visual Studio? I'd recommend against that for editing code and using a text editor like VS Code instead.